modules/installation-configuration-parameters.adoc

```diff
@@ -104,6 +105,11 @@ Before you deploy an {product-title} cluster, you provide parameter values to de
 After installation, you cannot modify these parameters in the `install-config.yaml` file.
 ====
 
+[IMPORTANT]
+====
+The `openshift-install` command does not validate optional parameters. If an incorrect parameter is specified, the related file or object is not created and no error is reported. Ensure that any optional parameters that are specified are correct.
```

Comment:
This is not quite correct. There is no validation on field names (optional or required). However, there is still validation on the values of optional fields.

Reply:
ACK - revised wording just pushed